src/components/attendee/VideoHuddle/useVideoHuddle.tsx

```diff
@@ -1,24 +1,103 @@
-import { useContext } from "react";
+import React, { useContext, useMemo, useState } from "react";
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
 
-import { VideoHuddleContext } from "./VideoHuddle";
+import { useVideoComms } from "../VideoComms/hooks";
+import { VideoTrack } from "../VideoComms/types";
+
+interface HuddleContextType {
```

Comment:
Yes, it is a type :) 

```suggestion
interface HuddleContext {
```